CHANGELOG.md

```diff
@@ -6,7 +6,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-[Unreleased]: https://github.com/envato/unwrappr/compare/v0.5.0...HEAD
+[Unreleased]: https://github.com/envato/unwrappr/compare/v0.6.0...HEAD
+
+## [0.6.0] 2021-05-12
+
+### Add
+- Allow specification of Gemfile lock files to annotate. ([#80])
```

Comment:
```suggestion
- Allow specification of Gemfile lock files to annotate. ([#86])

[0.6.0]: https://github.com/envato/unwrappr/compare/v0.5.0..v0.6.0
[#86]: https://github.com/envato/unwrappr/pull/86
```

Reply:
Thanks! I forgot to update it after creating the PR 😊 .